pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -54,28 +57,21 @@ const (
 	defaultSecondaryInterfaceName = "eth1"
 	startIfaceIndex               = 1
 	endIfaceIndex                 = 101
-)
 
-// Set resyncPeriod to 0 to disable resyncing.
-const resyncPeriod = 0 * time.Minute
+	interfaceDefaultMTU = 1500
```

Comment:
instead of a hardcoded 1500 value, should this be based on the MTU of the physical interface which we attach to the secondary OVS bridge?

Reply:
Yes, but if you do not mind, I plan to change this in a separate PR. Added a TODO.

Reply:
sounds good